### DISCUSSION TITLE
Insight Editor Issue: regex filtering?

### DISCUSSION THREAD

**Discussion by ExecutiveFunctioning** [TelemetryDeck/Viewer#306] [General]
I'm having an issue while editing an insiqht.

Insight ID: 637409F9-582F-46FA-8DD7-BB025FDD8B7A

I'm getting overly complex data about load times, and I'd like to simplify it so it's just the first integer after Optional, otherwise this chart shows garbage. I want to see the results grouped by integer, and clearly the way I'm using regex here is not correct. Is there another way?

<img width="598" alt="CleanShot 2023-10-04 at 14 33 31@2x" src="https://github.com/TelemetryDeck/Issues/assets/122486932/8d7b2817-b3a6-438d-bd97-b61fb337a5d2">

**Comment by winsmith** [comment]
This is possible using a [regular expression extraction function](https://telemetrydeck.com/docs/tql/extractionFunction/). Unfortunately the editor doesn't support these, so you'll have to convert to a Custom Query first.
